lighthouse-core/audits/user-timings.js

```diff
@@ -103,9 +103,15 @@ class UserTimings extends Audit {
 
       const details = Audit.makeTableDetails(headings, tableRows);
 
+      /** @type {LH.Audit.Details.DebugData|undefined} */
+      let debugData;
       /** @type {string|undefined} */
       let displayValue;
       if (userTimings.length) {
+        debugData = {
+          type: 'debugdata',
+          userTimings,
```

Comment:
looked at this one. i dont think there's any useful data in the raw userTimings obj that we want.

![image](https://user-images.githubusercontent.com/39191/82087370-ce68d080-96a4-11ea-8363-651f85df752d.png)

we can drop it.